token/client/src/token.rs

```diff
@@ -430,11 +440,12 @@ where
 
         instructions.extend_from_slice(token_instructions);
 
-        let latest_blockhash = self
-            .client
-            .get_latest_blockhash()
-            .await
-            .map_err(TokenError::Client)?;
+        let latest_blockhash = self.nonce_blockhash.unwrap_or(
```

Comment:
could make this change in the if-let below, so we consistently do the option 3-tuple thing everywhere, gross as it is :wink:

Reply:
Yeah sure, I didn't want to duplicate all of the other places where it's used below, but you're right that it would be clearer